flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -164,6 +164,15 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
 
       args.getIfNotNull<Boolean>("sendClientReports") { options.isSendClientReports = it }
 
+      args.getIfNotNull<Map<String, Any>>("sdk") { sdk ->
+        val name = sdk["name"] as? String
+        val version = sdk["version"] as? String
```

Comment:
```suggestion
        val name = sdk["name"] as? String?
        val version = sdk["version"] as? String?
```

Not sure if this is needed, just wondering if name or version is nullable, it might throw.

Reply:
Not needed AFAIK, the cast with `as?` handles nil strings.

<img width="725" alt="Bildschirm­foto 2023-01-16 um 10 05 42" src="https://user-images.githubusercontent.com/3984453/212639436-0ef76a47-ac18-4b47-8328-800b2f4d2610.png">